From a2cd2f556702148e86fb89349d595c232db85fdc Mon Sep 17 00:00:00 2001
From: Pankaj Bansal <pankaj.bansal@nxp.com>
Date: Mon, 22 Apr 2019 16:43:13 +0530
Subject: [PATCH 531/741] arm64: dts: fsl: layerscape: fix warnings when
 compiling dts files

commit 4fef5325d26e3d34fefc2811fd66cd940c6e6a55 from
http://source.codeaurora.org/external/qoriq/qoriq-components/linux

when compiling dts file using DTC_FLAG='-@', the device tree compiler
reports these warnings:

Warning (alias_paths): /aliases: aliases property name must include
only lowercase and '-'

Fixed the node aliases to silence these warnings.

Signed-off-by: Pankaj Bansal <pankaj.bansal@nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 .../boot/dts/freescale/fsl-ls1043a-qds.dts    | 32 +++++++++----------
 .../boot/dts/freescale/fsl-ls1046a-qds.dts    | 26 +++++++--------
 2 files changed, 29 insertions(+), 29 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-ls1043a-qds.dts b/arch/arm64/boot/dts/freescale/fsl-ls1043a-qds.dts
index cf6d96a8d67c..7e5fcdb1cb38 100644
--- a/arch/arm64/boot/dts/freescale/fsl-ls1043a-qds.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-ls1043a-qds.dts
@@ -24,22 +24,22 @@
 		serial1 = &duart1;
 		serial2 = &duart2;
 		serial3 = &duart3;
-		sgmii_riser_s1_p1 = &sgmii_phy_s1_p1;
-		sgmii_riser_s2_p1 = &sgmii_phy_s2_p1;
-		sgmii_riser_s3_p1 = &sgmii_phy_s3_p1;
-		sgmii_riser_s4_p1 = &sgmii_phy_s4_p1;
-		qsgmii_s1_p1 = &qsgmii_phy_s1_p1;
-		qsgmii_s1_p2 = &qsgmii_phy_s1_p2;
-		qsgmii_s1_p3 = &qsgmii_phy_s1_p3;
-		qsgmii_s1_p4 = &qsgmii_phy_s1_p4;
-		qsgmii_s2_p1 = &qsgmii_phy_s2_p1;
-		qsgmii_s2_p2 = &qsgmii_phy_s2_p2;
-		qsgmii_s2_p3 = &qsgmii_phy_s2_p3;
-		qsgmii_s2_p4 = &qsgmii_phy_s2_p4;
-		emi1_slot1 = &ls1043mdio_s1;
-		emi1_slot2 = &ls1043mdio_s2;
-		emi1_slot3 = &ls1043mdio_s3;
-		emi1_slot4 = &ls1043mdio_s4;
+		sgmii-riser-s1-p1 = &sgmii_phy_s1_p1;
+		sgmii-riser-s2-p1 = &sgmii_phy_s2_p1;
+		sgmii-riser-s3-p1 = &sgmii_phy_s3_p1;
+		sgmii-riser-s4-p1 = &sgmii_phy_s4_p1;
+		qsgmii-s1-p1 = &qsgmii_phy_s1_p1;
+		qsgmii-s1-p2 = &qsgmii_phy_s1_p2;
+		qsgmii-s1-p3 = &qsgmii_phy_s1_p3;
+		qsgmii-s1-p4 = &qsgmii_phy_s1_p4;
+		qsgmii-s2-p1 = &qsgmii_phy_s2_p1;
+		qsgmii-s2-p2 = &qsgmii_phy_s2_p2;
+		qsgmii-s2-p3 = &qsgmii_phy_s2_p3;
+		qsgmii-s2-p4 = &qsgmii_phy_s2_p4;
+		emi1-slot1 = &ls1043mdio_s1;
+		emi1-slot2 = &ls1043mdio_s2;
+		emi1-slot3 = &ls1043mdio_s3;
+		emi1-slot4 = &ls1043mdio_s4;
 	};
 
 	chosen {
diff --git a/arch/arm64/boot/dts/freescale/fsl-ls1046a-qds.dts b/arch/arm64/boot/dts/freescale/fsl-ls1046a-qds.dts
index 3ce68841cf1e..004e35452448 100644
--- a/arch/arm64/boot/dts/freescale/fsl-ls1046a-qds.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-ls1046a-qds.dts
@@ -26,19 +26,19 @@
 		serial2 = &duart2;
 		serial3 = &duart3;
 
-		emi1_slot1 = &ls1046mdio_s1;
-		emi1_slot2 = &ls1046mdio_s2;
-		emi1_slot4 = &ls1046mdio_s4;
-
-		sgmii_s1_p1 = &sgmii_phy_s1_p1;
-		sgmii_s1_p2 = &sgmii_phy_s1_p2;
-		sgmii_s1_p3 = &sgmii_phy_s1_p3;
-		sgmii_s1_p4 = &sgmii_phy_s1_p4;
-		sgmii_s4_p1 = &sgmii_phy_s4_p1;
-		qsgmii_s2_p1 = &qsgmii_phy_s2_p1;
-		qsgmii_s2_p2 = &qsgmii_phy_s2_p2;
-		qsgmii_s2_p3 = &qsgmii_phy_s2_p3;
-		qsgmii_s2_p4 = &qsgmii_phy_s2_p4;
+		emi1-slot1 = &ls1046mdio_s1;
+		emi1-slot2 = &ls1046mdio_s2;
+		emi1-slot4 = &ls1046mdio_s4;
+
+		sgmii-s1-p1 = &sgmii_phy_s1_p1;
+		sgmii-s1-p2 = &sgmii_phy_s1_p2;
+		sgmii-s1-p3 = &sgmii_phy_s1_p3;
+		sgmii-s1-p4 = &sgmii_phy_s1_p4;
+		sgmii-s4-p1 = &sgmii_phy_s4_p1;
+		qsgmii-s2-p1 = &qsgmii_phy_s2_p1;
+		qsgmii-s2-p2 = &qsgmii_phy_s2_p2;
+		qsgmii-s2-p3 = &qsgmii_phy_s2_p3;
+		qsgmii-s2-p4 = &qsgmii_phy_s2_p4;
 	};
 
 	chosen {
-- 
2.17.1

